lib/logitech_receiver/base.py

```diff
@@ -189,6 +197,15 @@ def read(handle, timeout=DEFAULT_TIMEOUT):
 		return reply[1:]
 
 
+# sanity checks on  message and report incorrect sizes when debugging
+def debug_message(data) :
+	assert isinstance(data, bytes), (repr(data), type(data))
+	report_id = ord(data[:1])
```

Comment:
Since Python 2 support is dropped, this could also be `data[0]` right?

Reply:
I guess so.  I'm rather leery of changing code that works fine.

Reply:
Done